lisp/ess-help.el

```diff
@@ -894,5 +881,61 @@ other dialects)."
       (set-window-point (get-buffer-window buf) pos) ;; don't move window point
       buf)))
 
+(with-no-warnings
+  ;; We're just backporting here, don't care about compiler warnings
+  (defalias 'ess-display-buffer-reuse-mode-window
+    ;; TODO: Remove once we drop support for Emacs 25
+    (if (fboundp 'display-buffer-reuse-mode-window)
```

Comment:
I think we can have defalias and defun inside the if check. It's strange to defalias to a lambda.

Reply:
If we use a defun we have to name it twice, which looked stranger to me than having a lambda. Up to you, though. I don't think it's a big deal either way, hopefully most people will be on Emacs 26+ soon.

Reply:
> I guess there is no other way without this extra backport, right?

Not if we want to keep `ess-display-buffer` fairly simple (or sacrifice reusing the help window on Emacs 25). I didn't realize `display-buffer-reuse-mode-window` didn't exist on Emacs 25 until I had finished refactoring and was testing. Luckily the backport only involved this single function that works on Emacs 25. It's simple enough to remove once we drop support for 25.

Reply:
All looks good. Thanks!